core/src/main/java/io/questdb/cairo/VarcharTypeDriver.java

```diff
@@ -258,12 +332,12 @@ public static Utf8Sequence getValue(long rowNum, MemoryR dataMem, MemoryR auxMem
         boolean isAscii = hasAsciiFlag(raw);
 
         if (hasInlinedFlag(raw)) {
-            // inlined string
             int size = (raw >> HEADER_FLAGS_WIDTH) & INLINED_LENGTH_MASK;
-            return ab == 1 ? auxMem.getVarcharA(auxOffset + 1, size, isAscii) : auxMem.getVarcharB(auxOffset + 1, size, isAscii);
+            return ab == 1
+                    ? auxMem.getVarcharA(auxOffset + 1, size, isAscii)
```

Comment:
nit: we should probably use `FULLY_INLINED_STRING_OFFSET` here and on the next line.